cypress.config.ts

```diff
@@ -72,7 +72,7 @@ export default defineConfig({
   },
   env: {
     baseUrl,
-    coverage:     false,
+    coverage:     true,
```

Comment:
Once again this is turned to `true`. Doing so it wastes time while developing E2E tests.
Please try to understand this point.